assets/styles/_default.scss

```diff
@@ -2,7 +2,7 @@
 // Inter UI and style overrides
 
 .nta-font {
-  font-family: "nta", Arial, sans-serif!important;
+  font-family: 'nta', Arial, sans-serif !important;
```

Comment:
what's with the `!important`s? do we know what we are overriding?

Reply:
not sure, but I didn't want to break the styles in this PR. If we decide to try to remove the `!important`'s then I think that should be a separate ticket.

For now I've just disabled the `no-important` rule.

Reply:
sure, that's me not reading the commit message or PR title. If we are rewriting this stuff we should bin the `!important'`s if we can, it can cause problems downstream